go/oasis-test-runner/oasis/fixture.go

```diff
@@ -118,7 +128,8 @@ func (f *ValidatorFixture) Create(net *Network) (*Validator, error) {
 	}
 
 	return net.NewValidator(&ValidatorCfg{
-		Entity: entity,
+		Entity:        entity,
+		SentryIndices: f.Sentries,
```

Comment:
Any reason why we don't resolve the indices here as we do with other things?

Reply:
Yes, makes sense. I've moved the sentry node resolution to `fixture.go`.